cmd/clef/extapi_changelog.md

```diff
@@ -10,6 +10,64 @@ TL;DR: Given a version number MAJOR.MINOR.PATCH, increment the:
 
 Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
 
+### 6.1.0
+
+The API-method `account_signGnosisTx` was added. This method takes two parameters, 
```

Comment:
I would probably include the name `Safe` since Gnosis does a lot of stuff :grimacing: